### PR TITLE
New file structure for Carmin Server

### PR DESCRIPTION
**Pull request by simon-dube**
This new file structure is based on Flask good practices mixed with the good elements of the Swagger code-gen from the OpenAPI 2.0 version.

### CODE REVIEW

File: carmin-server/cbrain/server/server.py [hunk 1]
```diff
@@ -1,17 +0,0 @@
-"""
-Launching Flask
-"""
-import os
-from flask import Flask
-
-APP = Flask(__name__)
```
Comment (louis-ver) [inline carmin-server/cbrain/server/server.py]: You can use lower-case for Flask decorators, as discussed.

**Review by louis-ver — APPROVED** [review]
LGTM

**Comment by glatard** [comment]
LGTM too!

**Comment by simon-dube** [comment]
@louis-ver Can you merge your changes to the Pylint test before I merge? I would like to merge your changes in my branch to merge only when the tests will pass on my end.

**Comment by simon-dube** [comment]
Checks have been done and pylint policy loosen. Merging!

**Pull request merged**